cocos/ui/editbox/edit-box-impl-base.ts

```diff
@@ -37,6 +37,11 @@ export class EditBoxImplBase {
      */
     public _delegate: EditBox | null = null;
 
+    /**
+     *  dirty flag to update the matrix
+     * */
```

Comment:
```suggestion
     */
```
Keep annotation format consistent